utils/docker-ce/files/dockerd.init

```diff
@@ -3,12 +3,83 @@
 USE_PROCD=1
 START=25
 
+DOCKERD_CONF="/tmp/dockerd/daemon.json"
+
+json_add_array_string() {
+	json_add_string "" "$1"
+}
+
+process_config() {
```

Comment:
I think it makes sense for `process_config()` to either return the config file or set `DOCKERD_CONF` directly, rather than using symlinks.

if `/etc/config/dockerd` doesn't exist `DOCKERD_CONF` equals `""` because the daemon will just startup with its defaults and not mislead anyone looking at the process command line.

if `alt_config_file` is set in the UCI, `DOCKERD_CONF` will equal that if it exists or `""` because it's most likely an error.

otherwise, we set `DOCKERD_CONF` to the one the UCI will generate `"/tmp/dockerd/daemon.json"`